File: FundamentosPython/Aula 03 - Condicoes/metodoPagamento.py
# Valor gasto pelo usuario.
# Metodo de pagamento
#   - Dinheiro/Pix = 10% desconto
#   - Credito = Valor total  

valorGasto = int(input("Informe o valor gasto: "))
print("INFORME O MÉTODO DE PAGAMENTO")
print("1 - DINHEIRO/PIX")
print("2 - CRÉDITO")
metodoPagamento = input("Digite o método de pagamento: ")

if metodoPagamento == "1":   
    valorFinal = valorGasto * 0.9
    print(f"O valor a ser pago é de R$ {valorFinal}")
else :
    print(f"O valor a ser pago é de R$ {valorGasto}")
    

File: DesenvolvimentoJavaScript/Aula10-EstruturasCondicionais/JavaScriptCodigo/JavaScript/metodoPagamento.js
// Valor gasto pelo usuario.
// Metodo de pagamento
//   - 1 - Boleto/Pix = 10% desconto
//   - 2 - Credito = Valor total  

let valorGasto = 1000
let metodoPagamento = 1

if (metodoPagamento == 1) {
    let valorFinal = valorGasto * 0.9
    console.log("MÉTODO DE PAGAMENTO BOLETO/PIX")
    console.log("O valor final foi de R$ " + valorFinal);
}else {
    console.log("MÉTODO DE PAGAMENTO CRÉDTIO")
    console.log("O valor final foi de R$" + valorGasto);

}